src/_test_/Header.test.js

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import Header from "../components/Header";
+import '@testing-library/jest-dom';
+
+describe("<Header />", () => {
+    it("renders without crashing", () => {
+    render(
+    <BrowserRouter>
+        <Header />
+    </BrowserRouter>
+    );
+
+    const component = document.querySelector("#header");
```

Comment:
You got the job done. I recommend thinking about how the user would access elements on the UI. Which is the goal of the react testing library. I am including an article that might help in this circumstance of some good practices with this library RTL.
https://medium.com/smallcase-engineering/why-should-we-not-use-document-queryselector-while-testing-react-components-835bbf34169e